examples/tinywl/XdgStackWorkspace.qml

```diff
@@ -154,6 +155,34 @@ Item {
 
                     waylandSurface.setMaximize(false)
                 }
+
+                function onRequestMinimize() {
+                    if (waylandSurface.isResizeing)
+                        return
+
+                    if (waylandSurface.isMinimize)
+                        return
+
+                    focus = false;
+                    Helper.setActivateSurface = null;
+
+                    waylandSurface.setMinimize(true)
+                    surface.minimize = true;
+                }
+
+                function onRequestCancelMinimize() {
```

Comment:
How cancel minimize in this example?

Reply:
need wlr_foreign_toplevel_manager/wlr_layer_shell for wlroots base dock/panal, cann't not test cancel minimize now


This pr needs to hang for a while

Reply:
We can add a simple dock in this example.